src/filters/utils.js

```diff
@@ -1,50 +1,158 @@
+example: |
+  {{ my_data | typeCheck('array') }}
+params:
+  val:
+    type: any
+    note: The data to check
+  is:
+    type: string
+    default: undefined
+    note: Optionally check for a particular type
+*/
 const typeCheck = (val, is) => {
   const type = typeof val;
   return is ? type === is : type;
 };
 
-const objectKeys = (obj) => Object.keys(obj);
-
-const slugify = (text) =>
-  text
-    .toString()
-    .toLowerCase()
-    .replace(/\s+/g, '-') // Replace spaces with -
-    .replace(/[^\w-]+/g, '') // Remove all non-word chars
-    .replace(/--+/g, '-') // Replace multiple - with single -
-    .replace(/^-+/, '') // Trim - from start of text
-    .replace(/-+$/, ''); // Trim - from end of text
-
-const unique = (array) =>
-  array.filter((value, index, self) => self.indexOf(value) === index);
+/* @docs
+label: get
+category: Inspect
+note: |
+  Returns true if an object contains a particular attribute
+  or attr:value pair,
+  including values inside an array.
+example: |
+  {% if page.data | get('author', 'miriam') %}
+params:
+  obj:
+    type: object
+  attrs:
+    type: string | array
+    note: Attribute to filter for
+  value:
+    type: any
+    default: undefined
+    note: Optional value to find within the attribute
+*/
+const get = (obj, attr, value) => {
+  // turn obj to arr for easier access...
+  const data = [].concat(obj);
+  const results = _.find(data, attr);
+  if (value !== undefined) {
+    return Object.values(results).includes(value);
+  }
+  return Boolean(results);
+};
 
-const jsonString = (item) => JSON.stringify(item);
+/* @docs
+label: just
+category: Data
+note: |
+  Returns a filtered array of objects with a given attribute,
+  or the first object where that attribute is equal to a particular value.
+example: |
+  {{ embed.figure(media | just('audio')) }}
+  {{ quotes.blockquote(press | just('slug', 'extension')) }}
+params:
+  array:
+    type: array of objects
+  attrs:
+    type: string | array
+    note: Attribute to filter for
+  value:
+    type: any
+    default: undefined
+    note: Optional attribute-value to just
+*/
+const just = (arr, attr, value) => {
+  const items = _.keyBy(arr, attr);
+  if (value !== undefined) {
+    return items[value];
+  }
+  return items;
+};
 
-const only = (list, start, end) => list.slice(start, end);
+/* @docs
+label: getJust
+category: Data
+note: |
+  The built-in nunjucks `slice` filter returns
+  an array of arrays at a given length.
+  This filter acts more like the js `array.slice()`,
+  returning a single sub-array from a given start index,
+  to the given end (or end of array).
+example: |
+  {# the first 5 items in a list #}
+  {% for item in my_array | items(0, 5) %}
+params:
+  data:
+    type: object
+  attrs:
+    type: string | array
+    note: The attributes to get
+  test:
+    type: object
+    note: attribute/value pairs to find in the resulting array
+*/
+const getJust = (data, attrs, test) => {
```

Comment:
@mirisuzanne tempted to delete this util since based on the description, it does the same as the `items` filter. 
 it's used in a couple places so left as is, so may need more info on what we'd like to accomplish with this one before refactoring.